app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/AddPictureActivityTest.kt

```diff
@@ -57,12 +58,14 @@ class AddPictureActivityTest {
             }
         }
 
+    private class testCat(override val name: String) : Category
```

Comment:
This class is identical to [model.DummyCategory](https://github.com/HumanLearning2021/HumanLearningApp/blob/91a17c46e68112645a37c40469a0b58aaa684c4a/app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/model/DummyCategory.kt) in #45, (and de facto identical in the current main) it might be less redundant to just use that.

Reply:
Indeed, but since these are tests I wanted them to be as independent as possible from other implementations.

Reply:
To me, one of the features of dummy implementations is writing tests, but I understand your reasoning. Resolved